dev/tests/acceptance/.env

```diff
@@ -1,6 +1,7 @@
-MAGENTO_BASE_URL=https://magento-venia.now.sh/
+MAGENTO_BASE_URL=https://release-dev-231-npzdaky-zddsyhrdimyra.us-4.magentosite.cloud/
```

Comment:
Discussed in person, but just to document. Rename to `.env.dist` and add step to docs saying to copy this file. Make sure you add `.env` to the `.gitignore` too.